### PR TITLE
Added CloudFormation Templates for sns-topic-publisher

### PR DESCRIPTION
**Pull request by stephensalim**
Added CFN Templates in JSON and YML format to for sns topic publisher

### CODE REVIEW

File: sns-topic-publisher/cfn-templates/sns-topic-publisher.json [hunk 1]
```diff
@@ -0,0 +1,175 @@
+{
+  "AWSTemplateFormatVersion": "2010-09-09",
+  "Parameters": {
+    "SNSTopicName": {
+      "Type": "String",
+      "Description": "Please enter your SNS Topic Name"
```
Comment (jaymccon) [inline sns-topic-publisher/cfn-templates/sns-topic-publisher.json]: Add a note to the description that SNS topic must be in the same region as the template

File: sns-topic-publisher/cfn-templates/sns-topic-publisher.json [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+              "Effect": "Allow",
+              "Resource": "arn:aws:logs:*:*:*"
+            },
+            {
+              "Sid": "Stmt1484080345748",
+              "Action": [
+                "sns:Publish"
+              ],
+              "Effect": "Allow",
+              "Resource": "*"
```
Comment (jaymccon) [inline sns-topic-publisher/cfn-templates/sns-topic-publisher.json]: provide publish permissions only to the specific topic

File: sns-topic-publisher/cfn-templates/sns-topic-publisher.json [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+          "ZipFile": {
+            "Fn::Join": [
+              "",
+              [
+                "// Sample Lambda Function to send notifications to a SNS topic when an AWS Health event happens\n",
+                "var AWS = require('aws-sdk');\n",
+                "var sns = new AWS.SNS();\n",
+                "\n",
+                "// define configuration\n",
+                "const snsTopic ='arn:aws:sns:",
```
Comment (jaymccon) [inline sns-topic-publisher/cfn-templates/sns-topic-publisher.json]: move configuration from function code into environment variables

**Comment by stephensalim** [comment]
Thanks for the suggestions!
Will update and request another pull :)

**Pull request closed without merging**